src/ReactiveUI.Validation/ValidationTexts/Abstractions/IValidationText.cs

```diff
@@ -5,7 +5,7 @@
 
 using System.Collections.Generic;
 
-namespace ReactiveUI.Validation.Collections;
+namespace ReactiveUI.Validation.ValidationTexts.Abstractions;
```

Comment:
The approval tests aren't passing due to namespace changes like this one. You should update the approved API files and the CI checks will pass then: https://github.com/reactiveui/ReactiveUI.Validation/tree/main/src/ReactiveUI.Validation.Tests/API